packages/components/src/components/form/Select/index.tsx

```diff
@@ -132,9 +137,15 @@ const Select = ({
     width,
     variant = 'large',
     noTopLabel = false,
+    useKeyPressScroll = false,
     ...props
 }: Props) => {
+    const selectRef: any = React.useRef(null);
     const theme = useTheme();
+
+    // if useKeyPressScroll flag is present, make sure isSearchable is false
+    isSearchable = useKeyPressScroll ? false : isSearchable;
```

Comment:
hmm this could be probably handled by some typescript magic (props could be union of 2 types, one type would include something like `useKeyPressScroll: true; isSearchable: never`, the other type would be `isSearchable: true; useKeyPressScroll: never `). I am not completely sure on the syntax... anyway this is just nitpicking, but if you would like to try it then go ahead :)